libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/JSONUtils.java

```diff
@@ -25,6 +24,10 @@
      * traverse the object graph and pull out the desired property
      */
     public static <U> U queryJSON(JSONObject source, String query, U defaultObject) {
+        if (source == null) {
```

Comment:
Since we're checking for a null `source`, should we also check for a null `query`?